crt_portal/static/sass/custom/intake.scss

```diff
@@ -116,12 +117,16 @@
 
     .crt-dropdown {
       margin-right: 2rem;
-      margin-bottom: 2rem;
+      margin-bottom: 1rem;
+      margin-top: 1rem;
```

Comment:
Did somebody request this change? or, is this one of the style change Haseeb made?

Reply:
These are changes that fix the padding issues between the 'Incident Location' dropdown and the 'Apply Filters' button. I figured it was a small fix and while I am not changing the 'Incident location' filter specifically, this helped the feel of the overall body filters. I can change this back if styling should be done at another time/card